docs/errors/comms.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Comms errors
+description: An index of Clerk errors related to comms.
```

Comment:
what is comms?

I see these errors are about Identification. Can we come up with a more specific title for this page?

Reply:
🤷🏻 https://github.com/clerk/clerk_go/blob/3df6867bfb2913c1ab95131a2bf49dee29d4c0c3/api/apierror/comms.go